packages/math-input/src/components/keypad-button.tsx

```diff
@@ -267,8 +265,6 @@ const focusBoxZIndex = 0;
 
 const styles = StyleSheet.create({
     buttonBase: {
-        // HACK(benkomalo): support old style flex box in Android browsers
-        "-webkit-box-flex": "1",
```

Comment:
TS wouldn't let this be here and since I think flexbox is pretty well supported I didn't feel like arguing.